example/pom.xml

```diff
@@ -89,6 +89,9 @@
               <value>LOC</value>
             </systemProperty>
           </systemProperties>
+          <webAppConfig>
```

Comment:
Can you remove this, or is it absolutely necessarily?

Reply:
It is not necessary, so it will be deleted.